garrysmod/lua/vgui/dhtml.lua

```diff
@@ -99,8 +99,8 @@ function PANEL:ConsoleMessage( msg, file, line )
 	--
 	-- Plain ol' console.log
 	--
-	MsgC( Color( 255, 160, 255 ), "[HTML] " )
-	MsgC( Color( 255, 255, 255 ), msg, "\n" )
+	MsgC( color_white, "[HTML] " )

```

Comment:
```suggestion
	MsgC( color_purple, "[HTML] " )
```

Define `color_purple = Color( 255, 160, 255 )` somewhere else - https://github.com/Facepunch/garrysmod/pull/1709#issuecomment-674349833